tests/ui/apps/VirtualList/VirtualGridList/VirtualGridList-View.js

```diff
@@ -0,0 +1,201 @@
+import spotlight from '@enact/spotlight';
+import SpotlightContainerDecorator from '@enact/spotlight/SpotlightContainerDecorator';
+import {Row, Column, Cell} from '@enact/ui/Layout';
+import ri from '@enact/ui/resolution';
+import React from 'react';
+
+import {Button} from '../../../../../Button';
+import Input from '../../../../../Input';
+import ImageItem from '../../../../../ImageItem';
+import {VirtualGridList} from '../../../../../VirtualList';
+import ThemeDecorator from '../../../../../ThemeDecorator';
+
+const ListContainer = SpotlightContainerDecorator('div');
+const OptionsContainer = SpotlightContainerDecorator({leaveFor: {down: '#left'}}, 'div');
+const getScrollbarVisibility = (hidden) => hidden ? 'hidden' : 'visible';
+
+// NOTE: Forcing pointer mode off so we can be sure that regardless of webOS pointer mode the app
+// runs the same way
+spotlight.setPointerMode(false);
+
+const items = [];
+
+// eslint-disable-next-line enact/prop-types, enact/display-name
+const renderItem = ({index, ...rest}) => {
+	const {source, subText, text} = items[index];
+	return (
+		<ImageItem
+			{...rest}
+			id={`item${index}`}
+			label={subText}
+			src={source}
+			style={{margin: 0}}
+		>
+			{text}
+		</ImageItem>
+	);
+};
+
+const updateData = (dataSize, noLabel) => {
+	const
+		itemNumberDigits = dataSize > 0 ? ((dataSize - 1) + '').length : 0,
+		headingZeros = Array(itemNumberDigits).join('0');
+
+	items.length = 0;
+
+	for (let i = 0; i < dataSize; i++) {
+		const
+			count = (headingZeros + i).slice(-itemNumberDigits),
+			text = `Item ${count}`,
+			subText = `SubItem ${count}`,
+			color = Math.floor((Math.random() * (0x1000000 - 0x101010)) + 0x101010).toString(16),
+			source = {
+				'hd': `http://placehold.it/200x200/${color}/ffffff&text=Image ${i}`,
+				'fhd': `http://placehold.it/300x300/${color}/ffffff&text=Image ${i}`,
+				'uhd': `http://placehold.it/600x600/${color}/ffffff&text=Image ${i}`
+			};
+
+		if (noLabel) {
+			items.push({source});
+		} else {
+			items.push({text, subText, source});
+		}
+	}
+
+	return dataSize;
+};
+
+class app extends React.Component {
+	constructor (props) {
+		super(props);
+		this.state = {
+			hideScrollbar: false,
+			horizontal: false,
+			noLabel: false,
+			numItems: 100,
+			minHeight: 450,
```

Comment:
ImageItem's height is needed to adjust. 
Our samplers are made on a monstone basis, so we have to make them to fit the FHD (1920X1080) resolution. When minHeight 450 is applied to  the height value in "style={{height: ri.scaleToRem(minHeight * 3)}" becomes too large to display all imageItem on one screen and is truncated.

Reply:
Hello @dongsuwon , I updated the view with proper min/height for ImageItem